src/main/java/xyz/rc24/bot/commands/wii/CodeCmd.java

```diff
@@ -242,7 +242,7 @@ protected void execute(CommandEvent event)
             List<String> args = parseArgs(REMOVE_PATTERN, event.getArgs());
             if(args.size() < 2)
             {
-                event.replyError("Wrong format! Correct one is `" + gs.getFirstPrefix() + "code remove <type> <name>");
+                event.replyError("Wrong format! Correct one is ``" + gs.getFirstPrefix() + "code remove <type> <name>``");
```

Comment:
Why double "`"

Reply:
I guess that does the same thing, not that it's a big deal

Reply:
Remove that, Discord is prone to breaking stuff easily